src/server/api/routes/user.router.js

```diff
@@ -14,6 +14,7 @@ const ROLES = require("../../constants/roles");
 const userController = require("../controllers/user.controller");
 
 // ENDPOINT: /api/users/ :GET
+
 router.get("/", authorizeUser(ROLES.SUPER_USER), (req, res, next) => {
```

Comment:
Did you intend to add these empty lines in a couple of files that are unrelated to your PR? ;)

Reply:
Do I need to delete the empty line?